### DISCUSSION TITLE
No audio in ubuntu LTS 22.10.

### DISCUSSION THREAD

**Discussion by tonmoy998** [i3/i3#5602] [Q&A]
I am using dirrect i3 without any destop environment.I am not getting audio in vlc,mpv .But Rhythbox working fine.And Youtube is not playing in browser.

**Comment by okraits** [comment]
I guess you should change the volume with some mixer application, pavucontrol for example, if you are using pulseaudio.

**Reply by tonmoy998** [reply]
I did with alsa mixer,pavucontrol but not working for me.But with sway it works

**Reply by okraits** [reply]
I doubt it has to do with i3 or sway because both are window managers which only care about windows and not about your sound daemon.
What is the difference between your i3 and sway config?

**Reply by tonmoy998** [reply]
here what I found
![Screenshot_২০২৩০৮০৫_১৯২৭০৯](https://github.com/i3/i3/assets/74363531/b1d9a92d-a7d6-4cc0-8781-0963d3f4c64f)
